ansible/group_vars/omero-hosts.yml

```diff
@@ -460,6 +460,15 @@ omero_web_apps_config_set:
       ns:
         - "openmicroscopy.org/omero/bulk_annotations"
       label: "Others"
+  - menu: "notebook"
+    config:
+      default:
+        - "Study Notebook"
+      all:
+        - "Study Notebook"
+      ns:
+        - "openmicroscopy.org/mapr/analysis/notebook"
```

Comment:
Trying to integrate this with recent efforts to expose to original analysis submitted by the authors. We have made extensive use of the `openmicroscopy.org/idr/analysis/original` namespace - see https://github.com/search?q=org%3AIDR+openmicroscopy.org%2Fidr%2Fanalysis%2Foriginal&type=code

Should this be `openmicroscopy.org/idr/analysis/notebook` instead?

Reply:
happy with the suggestion,  I will adjust the script

Reply:
script changes https://github.com/IDR/idr0093-mueller-perturbation/commit/aae814e2d3d6382312f4d4fdd1600690be74ffeb

cc @dominikl

Reply:
👍  Looks good to me.